thunder/tests/test_nvfuser.py

```diff
@@ -1061,3 +1061,18 @@ def test_scatter(executor, device: str, dtype: dtypes.dtype):
         assert len(fusion_bsyms) == 1
         outside_fusion_syms = ["unpack_trivial", "python_return"]
         assert {el.sym.name for el in fw_trace.bound_symbols if not el.sym.is_fusion} == set(outside_fusion_syms)
+
+
+@pytest.mark.xfail(reason="nvFuser does not support symbolic values for ceil")
+@instantiate(
+    executors=(nvFuserExecutor,),
+    dtypes=NOTHING,
+)
+def test_arange_symbolic_values(executor, device: str, dtype: dtypes.dtype):
+    from thunder.tests.opinfos import arange_opinfo
+
+    for sample in arange_opinfo.sample_inputs(device, thunder.float32):
+        compiled_func = executor.make_callable(torch.arange, cache="symbolic values")
+        out = compiled_func(*sample.args, **sample.kwargs)
```

Comment:
I think we should also add a a check to verify that there is only one bsym in trace and it is a nvfuser symbol.